### DISCUSSION TITLE
Any way to simulate clouds or gases in a Modelviewer scene ?

### DISCUSSION THREAD

**Discussion by anakin2022** [google/model-viewer#3726] [Q&A]
Hello!

I am designing a scene where clouds are needed.. do you know if there is any way to build clouds or objects that simulate gases and are compatible with modelviewer inside the glb file?

I tried to replicate the instructions in this video and create them but the clouds do not come out in the modelviewer editor.

www.youtube.com/watch?v=y-ofQpT0_EA

THANKS !!!

**Comment by elalish** [comment]
We do support the volume extension of glTF, but that's more for glass and such. It might look okay with a high roughness value and a gray absorption color, but not sure.
